eng/Versions.props

```diff
@@ -2,6 +2,10 @@
 <Project>
   <PropertyGroup Label="Version settings">
     <!-- This repo version -->
+    <!-- Workaround for https://github.com/microsoft/vstest/issues/4544 
+    Use semantic versioning V1 because V2 will produce version like 17.7.0-preview.23280.1+94103c3f and DTAExecutionHost 
+    is trying to parse that version and will consider any version with more than 4 `.` in it as invalid. -->
+    <SemanticVersioningV1>true</SemanticVersioningV1>
```

Comment:
Nit: can be skipped but I like the "WPF" format .

```suggestion
    <!-- 
        Workaround for https://github.com/microsoft/vstest/issues/4544 
        Use semantic versioning V1 because V2 will produce version like 17.7.0-preview.23280.1+94103c3f and DTAExecutionHost 
        is trying to parse that version and will consider any version with more than 4 `.` in it as invalid. 
    -->
    <SemanticVersioningV1>true</SemanticVersioningV1>
```

Reply:
Merged, it seems a bit too loud for something we won't be able to change any time soon. But I don't have a strong opinion.